scripts/pre-commit-check-code-generation

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Run GraphQL generation for Go files
+go generate ./... || exit
+
+# If in CirleCI environment, install goimports and specify absolute path to the binary
```

Comment:
Should we just add go imports to the project using `go.mod` so that it's already installed? Folks will have to do it locally, too, unless they only run it through `pre-commit`.